hack/toolchain.sh

```diff
@@ -17,7 +17,7 @@ tools() {
     go install github.com/norwoodj/helm-docs/cmd/helm-docs@v1.8.1
     go install sigs.k8s.io/controller-runtime/tools/setup-envtest@v0.0.0-20220421205612-c162794a9b12
     go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.8.0
-    go install github.com/sigstore/cosign/cmd/cosign@v1.9.0
+    go install github.com/sigstore/cosign/cmd/cosign@v1.10.0
```

Comment:
did you test this by making a snapshot release?

Reply:
Validated making a snapshot release with personal public ECR